engine.json.minimum

```diff
@@ -17,6 +17,9 @@
     "spark.kryoserializer.buffer.mb": "300",
     "spark.kryoserializer.buffer": "300m",
     "spark.executor.memory": "4g",
+    "spark.executor.cores": "2",
+    "spark.task.cpus": "2",
+    "spark.default.parallelism": "8",
     "es.index.auto.create": "true"
   },
   "algorithms": [
```

Comment:
I think we should create a set of tuning examples instead of putting this in `.minimum`, which was meant for the least number of params.

BTW I've heard of people trying larger numbers here by an order of magnitude, did you try any others?